src/pal/src/thread/thread.cpp

```diff
@@ -1420,6 +1421,10 @@ CorUnix::GetThreadTimesInternal(
         pTargetThread->Unlock(pThread);
         goto SetTimesToZero;
     }
+#else
+    /* Set dummy value */
+    cid = CLOCK_REALTIME;
```

Comment:
It would be better to return error here (`SetLastError(ERROR_INTERNAL_ERROR); goto SetTimesToZero;`) instead of returning a bogus value.